morse.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'morse_to_alphabet'
+
+def decode_character(character_in_morse_code)
+  MORSE_CODE[character_in_morse_code]
+end
+
+def decode_word(word_in_morse_code)
+  word_array = word_in_morse_code.split
+  message = []
+  word_array.each do |character|
+    message.push decode_character character
```

Comment:
* Kindly fix this unexpected behavior I get when  i try to run your code 
![image](https://github.com/danifromecuador/decode-morse-code/assets/52497246/dc87729e-587f-45d4-9055-997a94320b9f)

Reply:
Thank you, dear reviewer, the problem is solved, also I've changed the variable names and used string methods instead of array methods.